quick-start-with-tidb.md

```diff
@@ -38,16 +38,32 @@ As a distributed system, a basic TiDB test cluster usually consists of 2 TiDB in
     curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
     ```
 
+    Verify that TiUP has installed successfully by checking the output in the terminal or shell window for the following message:
+
+    ```log
+    Successfully set mirror to https://tiup-mirrors.pingcap.com
+    Detected shell: zsh
+    Shell profile:  /Users/user/.zshrc
+    /Users/user/.zshrc has been modified to add tiup to PATH
+    open a new terminal or source /Users/user/.zshrc to use it
+    Installed path: /Users/user/.tiup/bin/tiup
+    ===============================================
+    Have a try:     tiup playground
+    ===============================================
+    ```
+
+    Please pay attention to the Shell profile path above. After installation, this file has been modified to add tiup to PATH. You can open a new terminal or follow the next step to declare the global environment variables of the TiUP.
```

Comment:
1. Let's not overdo the politeness. ([ref](https://developers.google.com/style/tone))
2. "You can open a new terminal or follow the next step" might confuse the user.

```suggestion
    Note the Shell profile path in the output above. You need to use the path in the next step.
```